moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_display.cpp

```diff
@@ -936,6 +936,11 @@ void MotionPlanningDisplay::updateQueryGoalState()
   context_->queueRender();
 }
 
+void MotionPlanningDisplay::updatePreviousState()
+{
+  *previous_state_ = *query_start_state_->getState();
```

Comment:
How do you ensure that `previous_state_` is indeed a valid pointer?

Reply:
I thought it is ensured because ```previous_state``` is allocated before starting planning/execution and ```MotionPlanningFrame  (a caller of this function)``` calls this function only at the end of execution.
But, it can be change to call this function despite invalid between ```onInitialize()``` and ```onRobotModelLoaded()```.

Now I am thinking to add a guard  as below. 
```
if (previous_state_)
  return;

*previous_state_ = *query_start_state_->getState();
```
I would like to get your feedback.

Reply:
If you are sure that the pointer is always defined when getting here, that's fine.
In this case, just add a comment explaining why.

Reply:
I understand. I will add a comment.